projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json

```
@@ -82,50 +82,50 @@
+    "item.vintagedelight.gearo_berry_jam_bottle": "瓶装速速浆果酱",
+    "item.vintagedelight.ghost_pepper": "魔鬼椒",
     "item.vintagedelight.pickled_pepper": "泡椒",
-    "item.vintagedelight.ghost_pepper_seeds": "鬼椒种子",
-    "item.vintagedelight.pepper_jam_mason_jar": "泡椒酱罐",
-    "item.vintagedelight.pepper_jam_bottle": "泡椒酱瓶",
+    "item.vintagedelight.ghost_pepper_seeds": "魔鬼椒籽",
+    "item.vintagedelight.pepper_jam_mason_jar": "罐装辣椒酱",
+    "item.vintagedelight.pepper_jam_bottle": "瓶装辣椒酱",
     "item.vintagedelight.cucumber": "黄瓜",
     "item.vintagedelight.cucumber_seeds": "黄瓜种子",
     "item.vintagedelight.pickle": "酸黄瓜",
-    "item.vintagedelight.pickled_beetroot": "腌甜菜根",
-    "item.vintagedelight.pickled_onion": "腌洋葱",
-    "item.vintagedelight.pickled_egg": "腌蛋",
-    "item.vintagedelight.century_egg": "松花蛋",
-    "item.vintagedelight.relish_mason_jar": "黄瓜酱罐",
-    "item.vintagedelight.relish_bottle": "黄瓜酱瓶",
-    "item.vintagedelight.vinegar_mason_jar": "醋罐",
-    "item.vintagedelight.vinegar_bottle": "醋瓶",
-    "item.vintagedelight.oatmeal": "燕麦片",
-    "item.vintagedelight.oatmeal_cookie": "燕麦曲奇",
-    "item.vintagedelight.overnight_oats": "隔夜燕麦",
-    "item.vintagedelight.chocolate_nut_granola_bar": "巧克力坚果燕麦棒",
-    "item.vintagedelight.fruity_granola_bar": "果味燕麦棒",
-    "item.vintagedelight.deluxe_granola_bar": "豪华燕麦棒",
+    "item.vintagedelight.pickled_beetroot": "盐渍甜菜根",
+    "item.vintagedelight.pickled_onion": "盐渍洋葱",
+    "item.vintagedelight.pickled_egg": "咸蛋",
+    "item.vintagedelight.century_egg": "百年皮蛋",
+    "item.vintagedelight.relish_mason_jar": "罐装酸黄瓜酱",
+    "item.vintagedelight.relish_bottle": "瓶装酸黄瓜酱",
+    "item.vintagedelight.vinegar_mason_jar": "罐装醋",
+    "item.vintagedelight.vinegar_bottle": "瓶装醋",
+    "item.vintagedelight.oatmeal": "燕麦粥",
+    "item.vintagedelight.oatmeal_cookie": "燕麦饼干",
+    "item.vintagedelight.overnight_oats": "酒酿燕麦粥",
+    "item.vintagedelight.chocolate_nut_granola_bar": "士力架",
+    "item.vintagedelight.fruity_granola_bar": "果味士力架",
+    "item.vintagedelight.deluxe_granola_bar": "豪华士力架",
     "item.vintagedelight.mason_jar": "梅森罐",
-    "item.vintagedelight.apple_sauce_mason_jar": "苹果酱罐",
-    "item.vintagedelight.apple_sauce_bottle": "苹果酱瓶",
-    "item.vintagedelight.sweet_berry_mason_jar": "甜浆果果酱罐",
-    "item.vintagedelight.sweet_berry_jam_bottle": "甜浆果果酱瓶",
-    "item.vintagedelight.glow_berry_mason_jar": "发光浆果果酱罐",
-    "item.vintagedelight.glow_berry_jam_bottle": "发光浆果果酱瓶",
-    "item.vintagedelight.kimchi": "辣白菜",
-    "item.vintagedelight.pickle_soup": "腌菜汤",
+    "item.vintagedelight.apple_sauce_mason_jar": "罐装苹果酱",
+    "item.vintagedelight.apple_sauce_bottle": "瓶装苹果酱",
+    "item.vintagedelight.sweet_berry_mason_jar": "罐装甜浆果酱",
+    "item.vintagedelight.sweet_berry_jam_bottle": "瓶装甜浆果酱",
+    "item.vintagedelight.glow_berry_mason_jar": "罐装发光浆果酱",
+    "item.vintagedelight.glow_berry_jam_bottle": "瓶装发光浆果酱",
+    "item.vintagedelight.kimchi": "泡菜",
```

```suggestion
    "item.vintagedelight.kimchi": "辣白菜",
```
见https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/4617